util/shared/helpers.mjs

```diff
@@ -222,6 +223,8 @@ const localModules = (modulesFolder = '', markdownFolder = '', disableTransforms
     .reduce({}, fileCollectionReducer('/modules/'))
   }
 
+  const trimPath = file => file.startsWith(process.cwd()) ? file.substr(process.cwd().length + 1) : file
```

Comment:
It looks like maybe you're trying to trim the working directory off the front of the absolute file path. You can do that with node's `path` utility. For example
```js
const path = require('path');
const file = process.cwd() + '/some/path/to/file.js'
path.relative(process.cwd(), file) // some/path/to/file.js
```